airflow/dags/macros.py

```diff
@@ -106,9 +106,11 @@ def get_latest_schedule_data(table):
     return f"""
         SELECT
             * EXCEPT(calitp_deleted_at)
-        FROM gtfs_views_staging.{table}_clean
+        FROM gtfs_views_staging.{table}_clean t1
+        LEFT JOIN gtfs_schedule_history.calitp_feed_latest t2
+            USING(calitp_itp_id, calitp_url_number, calitp_extracted_at)
         WHERE
-            calitp_deleted_at = "2099-01-01"
+            t2.is_latest_load = true
```

Comment:
could you maybe include a comment that you are intentionally excluding tables without any corresponding `calitp_feed_latest` row

also don't need `= true`